boms/upperBoundsCheck/pom.xml

```diff
@@ -0,0 +1,84 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0
+                             http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>com.google.cloud</groupId>
+  <artifactId>zero-pom-test</artifactId>
+  <version>0.0.1-SNAPSHOT</version>
+  <packaging>pom</packaging>
+
+  <name>Cloud Platform Supported Libraries Upper Bounds Check</name>
+  <description>
+    Import the Cloud Platform Supported Libraries and verify that 
+    each library imported is the most recent of the versions
+    available in the dependency tree.
+  </description>
+
+  <organization>
+    <name>Google LLC</name>
+    <url>https://cloud.google.com</url>
+  </organization>
+
+  <licenses>
+    <license>
+      <name>The Apache License, Version 2.0</name>
+      <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
+    </license>
+  </licenses>
+
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+  </properties>
+
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>com.google.cloud</groupId>
+        <artifactId>zero-bom</artifactId>
+        <version>0.0.1-SNAPSHOT</version>
+        <type>pom</type>
+        <scope>import</scope>
+      </dependency>
+    </dependencies>    
+  </dependencyManagement>
+
+  <!-- TODO is there a way to run this test directly on the BOM
```

Comment:
Internally, I just wrote some details about [what I think we'd want](https://docs.google.com/document/d/1U2Ru2koluWfOXYskqW1NPHBp7uDcugpy23v1nR9M7DU/edit?ts=5b8493cd#heading=h.xhofic44mqoo). It's possible that we could do it in the BOM itself, but we'd have to write our own mojo, and even that might be tricky (though I don't really have any idea). Externally, I put together a hacky shell-based [proof of concept](https://docs.google.com/document/d/1HV-rfos5hMMzqix4orXiObEMTfBQefidN12boQw5U08/edit).